src/Toml/PrefixTree.hs

```diff
@@ -34,6 +36,9 @@ import qualified Data.List.NonEmpty as NonEmpty
 newtype Piece = Piece { unPiece :: Text }
     deriving (Show, Eq, Hashable)
 
+instance IsString Piece where
+    fromString = Piece . fromString
```

Comment:
I just made the same thing :slightly_smiling_face:  I just derived :)

Reply:
Yeah, I should derive it as well...